packages/release/etc-cni.mount

```diff
@@ -8,7 +8,7 @@ Before=local-fs.target umount.target
 What=tmpfs
 Where=/etc/cni
 Type=tmpfs
-Options=nosuid,nodev,noexec,noatime
+Options=nosuid,nodev,noexec,noatime,mode=0755
```

Comment:
Do we need a comment anywhere explaining why this one needs `0755` while everything else gets set to `0750`?

Reply:
`/etc/cni` is used by CNI plugins, where we don't necessarily know for sure what user containers in the pod will run with. The other directories should only be accessed by host processes.